src/GcsBackup.php

```diff
@@ -66,4 +87,18 @@ private function buildTreeFromPath(StorageObject $object, string $path): void
 
         $current[$filename] = $object->signedUrl(new DateTimeImmutable('+2 days'));
     }
+
+    private function createRetry(): RetryProxy
+    {
+        $backOffPolicy = new ExponentialBackOffPolicy(1000);
+        $retryPolicy = new CallableRetryPolicy(function (Throwable $e) {
+            if ($e instanceof RequestException && in_array($e->getCode(), self::RETRY_HTTP_CODES, true)) {
+                return true;
+            }
+
+            return false;
```

Comment:
Možná rovnou
```suggestion
            return $e instanceof RequestException && in_array($e->getCode(), self::RETRY_HTTP_CODES, true);
```
?

Reply:
může být -> udělám i v S3 backupu, ať je to stejně